.gitignore

```diff
@@ -21,6 +21,7 @@ dist/
 cudf.egg-info/
 python/build
 python/*/build
+python/dask-worker-space
```

Comment:
May want to make this recursive as well in case someone runs the pytest from within the `dask-cudf` or `custreamz` directory.